usr/share/rear/lib/filesystems-functions.sh

```diff
@@ -239,3 +239,8 @@ function xfs_parse
   # Output xfs options for further use
   echo "$xfs_opts"
 }
+
+# return the total used disk space of the target file systems
+function total_target_fs_used_disk_space() {
+    df --total --local -h --exclude-type=tmpfs --exclude-type=devtmpfs $(mount | grep "$TARGET_FS_ROOT" | cut -f 1 -d " ") | awk 'END{print $3}'
+}
```

Comment:
a line-break is missing here

Reply:
the newline at the end of the file is still missing here, cf.
https://github.com/rear/rear/pull/2973#issuecomment-1521648590